vendor/l.js

```diff
@@ -58,20 +54,13 @@
     ;
   //avoid multiple inclusion to override current loader but allow tag content evaluation
   if (!window.ljs) {
-    var checkLoaded = scriptTag.src.match(/checkLoaded/) ? 1 : 0
-      //-- keep trace of header as we will make multiple access to it
-      , urlParse = function (url) {
-        var parts = {}; // u => url, i => id, f => fallback, m => is a module
-        parts.u = url.replace(/(^module:)|#(=)?([^#]*)?/g, function (_, m, f, i) { parts[m ? 'm' : f ? 'f' : 'i'] = !!m || i; return ''; });
-        return parts;
-      }
+    //-- keep trace of header as we will make multiple access to it
+    urlParse = function (url) {
```

Comment:
I think you are missing a `var` here. This PR feels a bit like playing with fire to me...are you sure it's worth modifying this code for this small benefit of few KBs?

Reply:
Fair enough! I'll close it for now unless we see it's a problem